webapp/views.py

```diff
@@ -19,6 +30,197 @@ def get_cve(cve_id):
     return cve
 
 
+@use_kwargs(CVEsParameters, location="query")
+@marshal_with(CVEsAPISchema, code=200)
+def get_cves(**kwargs):
```

Comment:
I personally find this function quite difficult to review.
I wonder if splitting in functions would help. It looks like here you are: requesting releases, building the query for cves, and processing the response. Maybe splitting in this kind of logic would help to make it more readable?

Reply:
I have extracted out everything not related to applying filters and returning CVEs back.

The filters are still quite a complex chuck of code. Not sure how we could make it better, since the filter criteria are by themselves quite complex. But the function looks cleaner now imo.